README.md

```diff
@@ -18,2309 +18,93 @@ The Ethereum connector allows you to access the Ethereum JSON RPC API through Ba
-http:HttpConnectorError e;
-response, e = ethereumConnector.ethGetFilterChanges("2.0", 1999, "0x1bad3335bdd994b7a4690f0a134fb340");
-if (e == null) {
-    println(response);
-} else {
-    println(e);
-}
-```
-
-```json
-{
-  "id":1999,
-  "jsonrpc":"2.0",
-  "result": [{
-    "logIndex": "0x1",
-    "blockNumber":"0x5f9",
-    "blockHash": "0x8216c5785ac562ff41e2dcfdf5785ac562ff41e2dcfdf829c5a142f1fccd7d",
-    "transactionHash":  "0xdf829c5a142f1fccd7d8216c5785ac562ff41e2dcfdf5785ac562ff41e2dcf",
-    "transactionIndex": "0x0",
-    "address": "0x1bad3335bdd994b7a4690f0a134fb340f1fccd7d",
-    "data":"0x0000000000000000000000000000000000000000000000000000000000000000",
-    "topics": ["0x59ebeb90bc63057b6515673c3ecf9438e5058bca0f92585014eced636878c9a5"]
-    },{
-      
-    }]
-}
+import wso2/ethereum;
 ```
-##### Quick Testing
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_getFilterChanges 0x15842a94627e19e5571559236b5b1700`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getfilterchanges
-
-***
-
-#### ethGetFilterLogs
-
-Returns an array of all logs matching filter with given id.
-
-
-##### Parameters
 
-1. `string` - JSON RPC version
-2. `int` - network id
-3. `string` - The filter id.
-
-##### Returns
-
-See [ethGetFilterChanges](#ethgetfilterchanges)
-
-##### Quick Testing
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_getFilterLogs 0x15842a94627e19e5571559236b5b1700`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getfilterlogs
-
-***
-
-#### ethGetLogs
-
-Returns an array of all logs matching a given filter object.
-
-##### Parameters
-
-1. `string` - JSON RPC version
-2. `int` - network id
-3. `json` - the filter object, see [eth_newFilter parameters](#eth_newfilter).
-
-##### Returns
-
-See [ethGetFilterChanges](#ethgetfilterchanges)
-
-##### Quick Testing
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_getLogs '{"fromBlock": "0x1","toBlock": "0x2","address": "0x0eb8a07d29f5afdcbec1a9d087ece456139bfb87","topics":["0x000000000000000000000000a94f5374fce5edbc8e2a8697c15331677e6ebf0b", null,["0x000000000000000000000000a94f5374fce5edbc8e2a8697c15331677e6ebf0b", "0x0000000000000000000000000aff3454fce5edbc8cca8697c15331677e6ebccc"]]}'`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getLogs
-
-***
-
-#### ethGetWork
-
-Returns the hash of the current block, the seedHash, and the boundary condition to be met ("target").
-
-##### Parameters
-none
-
-##### Returns
-
-`json` - Array with the following properties:
-  1. `string`, 32 Bytes - current block header pow-hash
-  2. `string`, 32 Bytes - the seed hash used for the DAG.
-  3. `string`, 32 Bytes - the boundary condition ("target"), 2^256 / difficulty.
-
-##### Example
+In order for you to use the ethereum Connector, first you need to create an ethereum Client endpoint.
```

Comment:
Make sure that "ethereum" is in title case everywhere in the content. Change to "Ethereum". Apply elsewhere if applicable.